src/Stripe.net/Services/EphemeralKeys/StripeEphemeralKeyService.cs

```diff
@@ -12,6 +12,13 @@ public class StripeEphemeralKeyService : StripeBasicService<StripeEphemeralKey>
         // Sync
         public virtual StripeEphemeralKey Create(StripeEphemeralKeyCreateOptions createOptions, StripeRequestOptions requestOptions = null)
         {
+            // Creating an ephemeral key requires a specific API version to be set. This is handled as a parameter
+            // but has to be set on the StripeRequestOptions instead.
+            if (requestOptions == null)
```

Comment:
`requestOptions = requestOptions ?? new StripeRequestOptions();`

Reply:
oh neat I had never heard of this one but I can see it's used elsewhere in the library so will fix.